theano/tensor/nnet/nnet.py

```diff
@@ -751,7 +751,9 @@ def local_logsoftmax(node):
             isinstance(node.inputs[0].owner.op, Softmax)):
         inVars = node.inputs[0].owner.inputs[0]
         new_op = LogSoftmax()
-        return [new_op(inVars)]
+        ret = new_op(inVars)
+        ret.values_eq_approx = values_eq_approx_remove_inf
```

Comment:
I think it is better to set it under ret.tag.values_eq_approx. Tag are keep when cloning. Also this prevent to cluter the var itself. The interface you use is the old one.